.github/workflows/continuous_deployment.yml

```diff
@@ -0,0 +1,93 @@
+name: Tag Stable Release
+run-name: "CD for Release Branch ${{ github.ref_name }}"
+on:
+  push:
+    branches:
+      - 'stable/**'
+
+jobs:
+  get-latest-tag:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    outputs:
+      version: ${{ steps.bump_version.outputs.version || steps.init_version.outputs.version }}
+    steps:
+    - name: Extract Branch Version
+      shell: bash
+      run: |
+        # Strip the `stable/` prefix and the `x` suffix from the current
+        # release branch
+        VERSION_PREFIX=$(echo "${GITHUB_REF#refs/heads/}" | sed -e "s/^stable\///" -e "s/x//")
+        echo "version_prefix=${VERSION_PREFIX}" >> $GITHUB_OUTPUT
```

Comment:
defining bash in a yaml file feels weird to me, should we create scripts for the bash in this file?

Reply:
@chavesj I think your proposal makes sense, that these should not like in this file. I'm going to create a follow up ticket in the CI/CD work to look into turning these bash scripts into reusable github actions.

Reply:
Ticket https://klaviyo.tpondemand.com/entity/165151-create-a-github-action-for-tag